src/modules/drawing/DrawingModule.vue

```diff
@@ -119,7 +117,7 @@ function removeLastPointOnDeleteKeyUp(event) {
 }
 
 async function closeDrawing() {
-    isClosing.value = true
+    await store.dispatch('setShowLoadingBar', true)
```

Comment:
We should not have any spinner when closing the drawing like we used to have. We used to be able to close without any blocking action, this should still be then case. 
The original loading bar which was the red bar moving from right to left was much better as a spinner I think.

Reply:
I would separate this functionality from this PR and only push merge the zoom issue. And create a separate PR for the drawing spinner, for this one I think it requires some UI refinement/discussion. 
We should also try to keep the PR small and not mixing everything together, keeping bug fix in separate PR.

Reply:
It got mixed because I intended to use this backdrop/sprinner when going back 2D, but in the end I used the loading bar (as we discussed)
I can separate this into two work package, although I think this version of the close drawing is better than what we have on `develop`